api/storage_service.cc

```diff
@@ -76,6 +76,22 @@ static sstring validate_keyspace(http_context& ctx, const parameters& param) {
     throw bad_param_exception("Keyspace " + param["keyspace"] + " Does not exist");
 }
 
+static std::vector<sstring> validate_tables(const sstring& ks_name, http_context& ctx, const std::unordered_map<sstring, sstring>& query_params, sstring param_name) {
```

Comment:
nitpick: I would call it split_and_validate_tables or even parse_tables since it also does the splitting, not just validation.

Reply:
ok